OJS.Workers.ExecutionStrategies/PHP/PhpCliExecuteAndCheckExecutionStrategy.cs

```diff
@@ -56,10 +49,34 @@ public class PhpCliExecuteAndCheckExecutionStrategy : BaseInterpretedCodeExecuti
                 result.Results.Add(testResult);
             }
 
-            // Clean up
-            File.Delete(codeSavePath);
+            return result;
```

Comment:
This `foreach` can be refactored into a `.ForEach`

Reply:
Will not do this, because the submodule is used in judge.softuni.org where the package "MissingFeatures" is used also and ambiguous reference occurs, forcing us to use the extension method as static in some files e.g `ForEach(source, action)` instead of `source.ForEach(action)`